test/models/concerns/release_episodes_test.rb

```diff
@@ -0,0 +1,116 @@
+require "test_helper"
+
+class ReleaseEpisodesTest < ActiveSupport::TestCase
+  let(:episode) { create(:episode, published_at: 30.minutes.from_now) }
+  let(:podcast) { episode.podcast }
+  let(:feed) { podcast.default_feed }
+
+  describe ".to_release" do
+    it "returns episodes/podcasts that need release" do
```

Comment:
Apologies if these tests are a bit opaque.  I started out trying to set them up with static data, and then incrementing/stubbing `Time.now` to watch how things react over time.

But - since this relies on Postgres `NOW()`, that didn't really work out.  So I ended up modifying the other 2 variables: `episode.published_at` and `publishing_queue_item.created_at`.